Import web-platform-tests/html/semantics/popovers tests.

ziransun opened the pull request:
#### 0041f5bf29c3125d5ab371dc2e741239c8f18daa
<pre>
Import web-platform-tests/html/semantics/popovers tests.
<a href="https://bugs.webkit.org/show_bug.cgi?id=251907">https://bugs.webkit.org/show_bug.cgi?id=251907</a>

Reviewed by Tim Nguyen.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/hide-other-popover-side-effects-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/hide-other-popover-side-effects.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/idlharness.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/idlharness.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/light-dismiss-event-ordering.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/light-dismiss-event-ordering.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-change-display-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-change-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-change-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-display-none.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-display-none.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-display-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-idl-property.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-idl-property.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-multicol-display.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-multicol-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-nested-display-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-nested-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-nested-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-nesting.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-nesting.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-scroll-display-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-scroll-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-scroll-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-display-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-hide-cleanup.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-hide-cleanup.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-hide-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-hide-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-hide-finishes-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-hide-finishes.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-hide-finishes.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-show-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-show-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animation-corner-cases.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animation-corner-cases.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-appearance-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-appearance.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-appearance.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-attribute-basic.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-attribute-basic.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-backdrop-appearance-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-backdrop-appearance.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-backdrop-appearance.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-beforetoggle-opening-event.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-beforetoggle-opening-event.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-dialog-appearance-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-dialog-appearance.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-dialog-appearance.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-dialog-crash.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-document-open.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-document-open.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-events.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-events.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-focus-2.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-focus-2.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-focus-child-dialog-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-focus-child-dialog.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-focus.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-focus.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-hidden-display-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-hidden-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-hidden-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-inside-display-none-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-inside-display-none.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-inside-display-none.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-invoking-attribute.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-invoking-attribute.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-light-dismiss-on-scroll.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-light-dismiss-on-scroll.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-light-dismiss.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-light-dismiss.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-manual-crash.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-not-keyboard-focusable.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-not-keyboard-focusable.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-open-display-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-open-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-open-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-open-overflow-display-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-open-overflow-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-open-overflow-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-removal-2.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-removal-2.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-removal.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-removal.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-shadow-dom.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-shadow-dom.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-stacking-context-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-stacking-context.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-stacking-context.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-stacking.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-stacking.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-target-element-disabled.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-target-element-disabled.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-top-layer-combinations.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-top-layer-combinations.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-top-layer-interactions.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-types.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-types.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/resources/popover-styles.css: Added.
(.fake-popover):
(.fake-popover-backdrop):
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/resources/popover-utils.js: Added.
(waitForRender):
(async clickOn):
(async sendTab):
(async sendEscape):
(async sendEnter):
(isElementVisible):
(async finishAnimations):
(mouseOver):
(msSinceMouseOver):
(async waitForHoverTime):
(async blessTopLayer):
(showDefaultopenPopoversOnLoad.show):
(showDefaultopenPopoversOnLoad):
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/toggleevent-interface.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/toggleevent-interface.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/w3c-import.log: Added.
* LayoutTests/tests-options.json:

Canonical link: <a href="https://commits.webkit.org/260128@main">https://commits.webkit.org/260128@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/d21cfce87c75854a71f8243b5d4855a0e5c473f1

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/106896 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/15932 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/39695 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/116077 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/115629 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/17417 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/7130 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/99083 "Built successfully") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/112663 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/13184 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/96199 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/40779 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/95074 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/27844 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/82522 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/9082 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/29199 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/9659 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/6200 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/15249 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/48753 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/7001 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/11188 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
r=me with imported/w3c/web-platform-tests/html/semantics/popovers/popover-focus-2.tentative.html rebaselined per EWS

ziransun committed:
Import web-platform-tests/html/semantics/popovers tests.
https://bugs.webkit.org/show_bug.cgi?id=251907

Reviewed by Tim Nguyen.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/hide-other-popover-side-effects-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/hide-other-popover-side-effects.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/idlharness.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/idlharness.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/light-dismiss-event-ordering.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/light-dismiss-event-ordering.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-change-display-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-change-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-change-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-display-none.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-display-none.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-display-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-idl-property.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-idl-property.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-multicol-display.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-multicol-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-nested-display-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-nested-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-nested-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-nesting.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-nesting.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-scroll-display-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-scroll-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-anchor-scroll-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-display-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-hide-cleanup.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-hide-cleanup.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-hide-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-hide-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-hide-finishes-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-hide-finishes.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-hide-finishes.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-show-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animated-show-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animation-corner-cases.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-animation-corner-cases.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-appearance-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-appearance.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-appearance.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-attribute-basic.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-attribute-basic.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-backdrop-appearance-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-backdrop-appearance.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-backdrop-appearance.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-beforetoggle-opening-event.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-beforetoggle-opening-event.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-dialog-appearance-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-dialog-appearance.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-dialog-appearance.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-dialog-crash.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-document-open.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-document-open.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-events.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-events.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-focus-2.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-focus-2.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-focus-child-dialog-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-focus-child-dialog.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-focus.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-focus.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-hidden-display-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-hidden-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-hidden-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-inside-display-none-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-inside-display-none.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-inside-display-none.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-invoking-attribute.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-invoking-attribute.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-light-dismiss-on-scroll.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-light-dismiss-on-scroll.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-light-dismiss.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-light-dismiss.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-manual-crash.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-not-keyboard-focusable.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-not-keyboard-focusable.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-open-display-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-open-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-open-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-open-overflow-display-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-open-overflow-display.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-open-overflow-display.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-removal-2.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-removal-2.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-removal.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-removal.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-shadow-dom.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-shadow-dom.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-stacking-context-ref.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-stacking-context.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-stacking-context.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-stacking.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-stacking.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-target-element-disabled.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-target-element-disabled.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-top-layer-combinations.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-top-layer-combinations.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-top-layer-interactions.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-types.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-types.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/resources/popover-styles.css: Added.
(.fake-popover):
(.fake-popover-backdrop):
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/resources/popover-utils.js: Added.
(waitForRender):
(async clickOn):
(async sendTab):
(async sendEscape):
(async sendEnter):
(isElementVisible):
(async finishAnimations):
(mouseOver):
(msSinceMouseOver):
(async waitForHoverTime):
(async blessTopLayer):
(showDefaultopenPopoversOnLoad.show):
(showDefaultopenPopoversOnLoad):
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/toggleevent-interface.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/toggleevent-interface.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/w3c-import.log: Added.
* LayoutTests/tests-options.json:

Canonical link: https://commits.webkit.org/260128@main
